services/movie_session.py

```diff
@@ -42,3 +44,11 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get()` method is used without handling the `DoesNotExist` exception. Consider using `get_object_or_404()` or wrapping this call in a try-except block to handle the exception and avoid runtime errors.